### PR TITLE
Serve the proxy.html ourselves instead of proxying

### PR DESCRIPTION
**Pull request by inklesspen**
We shouldn't proxy to the old discovery API; for one thing, the proxy
approach doesn't work with different base paths.

### CODE REVIEW

File: endpoints/endpoints_dispatcher.py [hunk 1]
```diff
@@ -82,7 +86,7 @@ def __init__(self, backend_wsgi_app, config_manager=None):
       self._add_dispatcher('%sexplorer/?$' % base_path,
                            self.handle_api_explorer_request)
       self._add_dispatcher('%sstatic/.*$' % base_path,
```
Comment (tangiel) [inline endpoints/endpoints_dispatcher.py]: Mostly LGTM, but can't we just make this handler do just static/proxy.html? Removes the need to do a 404 handler.

Comment (inklesspen) [inline endpoints/endpoints_dispatcher.py]: I want to avoid sending that stuff to the backend and instead handle it within this file.

**Comment by tangiel** [comment]
Also update the proxy.html like this PR: https://github.com/cloudendpoints/endpoints-java/pull/100

**Comment by inklesspen** [comment]
@tangiel I've updated the proxy.html. If you're satisfied with this I'm going to rebase and merge.

**Pull request merged**